cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -385,16 +229,12 @@ private String extractMainTitle() {
                    .flatMap(Collection::stream);
     }
 
-    private CristinBookOrReportMetadata extractCristinBookReport() {
+    public CristinBookOrReportMetadata extractCristinBookReport() {
```

Comment:
This and the method extractCristinJournalPublication method are made public as they need to be reach from the new class PublicationInstanceBuilder